docs/content/gs-adding-users-proc.adoc

```diff
@@ -1,41 +1,39 @@
-
-image:../assets/images/gs-adding-users-sso-route.png[SSO route]
-
-- Click on the `Administration Console` link on the SSO landing screen.
-- Login to the SSO administration console as the `customer-admin` user.
-
-== Viewing Users in the SSO Admin Console
-- Navigate to the *Users* menu.
-- Click the *View all users* button at the top of the page. This will show the list of users currently available in the Integreatly environment.
+== Accessing the Red Hat Single Sign-On administration console
+. Login to the OpenShift console as the `customer-admin` user. 
+. Navigate to the *sso* project.
+. Retrieve the credentials for the Red Hat Single Sign-On administration console:
+.. Click *Secrets* in the *Resources* menu.
+.. Click *credential-rhsso* in the list of Secrets.
+.. Click *Reveal Secret* in the  *credential-rhsso* detail page.
+.. Note the values for *SSO_ADMIN_USERNAME* and *SSO_ADMIN_PASSWORD*.
+.. Click *Overview* to navigate back to the *sso* project overview page.
 
-image:../assets/images/gs-adding-users-sso-view-users.png[Viewing all users in SSO]
+. Click *Routes* in the *Applications* menu.
+. Click on the *sso* route hostname to open the Red Hat Single Sign-On (SSO) application.
+- Click on the *Administration Console* option on the Red Hat Single Sign-On landing page.
+- Login to the Red Hat Single Sign-On administration console.
 
+== Viewing Users in the Red Hat Single Sign-On administration console
 
-- The search box at the top of the page can be used to query the list and only show users that matches your criteria.
+By default, when you log into Red Hat Single Sign-On administration console, the *Openshift* realm is displayed.
 
-image:../assets/images/gs-adding-users-sso-search-users.png[Searching for users in SSO]
+. Navigate to the *Users* menu. The search box at the top of the page can be used to query the list and only show users that matches your criteria.
+. Click the *View all users* button at the top of the page. This displays the list of users currently available in the {PRODUCT} environment.
 
 == Creating Users in the SSO Admin Console
-- Navigate to the *Users* menu.
-- Click the *Add User* button at the top right corner of the page.
-
-image:../assets/images/gs-adding-users-sso-add-user.png[User form in SSO]
-
-- Enter the user information required in the user creation form.
-  * Ensure that the `Username` and `Email` fields are filled in.
-  * Ensure that field `User Enabled` is set to *ON*. This ensures that the user account will be activated and can be used for login.
-  * Ensure that the field `Email Verified` is set to *ON*. This ensures that the user account will be activated in 3Scale.
+. Navigate to the *Users* menu.
+. Click the *Add User* button at the top right corner of the page.
+. Enter the user information:
++
+ * Enter values to `Username` and `Email`.
+ * Set `User Enabled` to *ON*.
+ * Set `Email Verified` to *ON*. This ensures that the user account will be activated in 3Scale.
```

Comment:
@JameelB  is this line about 3scale true still?

Reply:
Yes you need to do that otherwise 3scale wont let the user login